server/email/email-router.js

```diff
@@ -0,0 +1,21 @@
+var express = require('express');
+var path = require('path');
+
+module.exports = function(app, io){
+  app.use(express.static(__dirname));
```

Comment:
is this express.static needed? seems like you serve up the only html from 'confirmation/confirmation.html'